api/handlers_ltm.go

```diff
@@ -64,6 +64,11 @@ func (s *server) ShowClientSSLProfile(w http.ResponseWriter, r *http.Request) {
 		handleError(w, err)
 	}
 
+	if out == nil {
+		handleError(w, apierror.New(apierror.ErrNotFound, fmt.Sprintf("%s not found", name), nil))
+		return
```

Comment:
Do you need a return after `handleError`?

Reply:
yes, handleError doesnt return from the handler

Reply:
Thanks guys.